lib/characters.js

```diff
@@ -0,0 +1,113 @@
+exports.eof = null
```

Comment:
What would be the need to have this file? Will all texts have only alphanumeric characters?
I also saw usage of `fromCharCode` and `charCodeAt`. Wouldn't it be preferable to acknowledge other character sets? (I'm thinking `fromCodePoint` and `codePointAt`)

Reply:
This file exists to have the codes used by markdown syntax. I copied it from babel, where they use a babel transform to compile them away and use the codes directly, as markdown only uses ASCII markers, we’re fine using charCodeAt I think!